index.html

```diff
@@ -141,7 +141,7 @@ <h2 class="d2l-heading-3">Components</h2>
 		<li>
 			Menus
 			<ul>
-				<li><a href="components/menu/demo/menu.html">d2l-menu</a></li>
+				<li><a href="components/menu/demo/menu.html?link-new-window-icon=true&link-click-changes=true">d2l-menu</a></li>
```

Comment:
Thought this could be a nice temporary way for someone to see flags available for a demo page. Just need to remember to remove it